dev/com.ibm.ws.microprofile.metrics.monitor_fat/fat/src/com/ibm/ws/microprofile/metrics/monitor_fat/TestEnableDisableFeaturesTest.java

```diff
@@ -178,6 +178,7 @@ public void testEDF2() throws Exception {
     	Assert.assertNotNull("No CWPMI2001I was found.", logMsg);
     	serverEDF2.setMarkToEndOfLog(serverEDF2.getMostRecentTraceFile());
        	Log.info(c, testName, "------- threadpool metrics should be available ------");
+       	Log.info(c, testName, serverEDF2.waitForStringInLog("defaultHttpEndpoint-ssl",60000));
```

Comment:
if the `defaultHttpEndpoint-ssl` does not show up, the test will continue.  You just see a `null` in the messages.log.  Please use `Assert.assertNotNull()`.